docs/_static/css/custom-theme.css

```diff
@@ -18,4 +80,155 @@
 
 .wy-nav-content {
     border-left: 2px #EEEEF1 solid;
-}
\ No newline at end of file
+    max-width: 1200px;
+}
+.caption {
+    border-top: 1px solid #eeeef1;
+}
+.caption-text {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:500;
+    font-size: 12px;
+    color: #666669;
+    line-height: 14px;
+    text-transform: capitalize;
+}
+
+.current {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:400;
+    font-size: 13px;
+    color: #555558;
+    line-height: 13px;
+    background: #fff;
+}
+
+.reference.internal {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:400;
+    font-size: 13px;
+    color: #555558;
+    line-height: 20px;
+    text-decoration: none;
+}
+.wy-menu-vertical a:hover {
+    background: #fff;
+    color: #e31b37;
+}
+.reference.internal.current {
+    color: #e31b37;
+}
+.wy-menu-vertical li.current {
+    background: #fff;
+}
+.wy-menu-vertical li.current a:hover {
+    background: #fff;
+    color: #e31b37;
+}
+.wy-menu-vertical li.toctree-l1.current>a {
+    border:none;
+}
+.wy-menu-vertical li.toctree-l2.current>a {
+    background: #fff;
+}
+.wy-menu-vertical li.toctree-l2.current li.toctree-l3>a {
+    background: #fff; 
+}
+.wy-menu-vertical li.toctree-l3.current li.toctree-l4>a {
+    background: #fff;
+}
+
+.wy-side-nav-search>a.icon img.logo, .wy-side-nav-search .wy-dropdown>a.icon img.logo {
+    width: 74px;
+    padding-left: 0;
+}
+.wy-side-nav-search input[type=text] {
+    font-family: Roboto,Arial,sans-serif;
+    border-color: #eeeef1;
+    border-radius: 3px;
+    background-color: #eeeef1;
+}
+
+.wy-side-nav-search>a, .wy-side-nav-search .wy-dropdown>a {
+    display: table-cell;
+}
+.wy-side-nav-search>a.icon img.logo, .wy-side-nav-search .wy-dropdown>a.icon img.logo {
+    margin-left: 0;
+    margin-top: 0;
+    width: 180px;
+    margin-bottom: 24px;
+}
+
+.wy-breadcrumbs {
+    display: none;
+}
+.rst-breadcrumbs-buttons {
+    display: none;
+}
+.rst-footer-buttons {
+    display: none;
+}
+
+
+
+
+.rst-content dl:not(.docutils) dt {
+    color: #3C96BA;
+    font-family: Roboto,Arial,sans-serif;
+    border-top: solid 3px #3C96BA;
+    background: rgba(60,150,186,0.15);
+}
+.rst-content .viewcode-link, .rst-content .viewcode-back {
+    color: #e31b37;
+}
+
+
+
+.highlight .kn {
+    color: #FF0079 !important;
+}
+.highlight .s1 {
+    color: #76B166 !important;
+}
+.highlight .kc {
+    color: #4AC6AA !important;
+}
+.highlight .n {
+    color: #444447 !important;
+}
+.highlight .mi {
+    color: #EB904B !important;
+}
+.highlight .gp {
+    color: #000003 !important;
+}
+.highlight .nn {
+    color: #322285 !important;
+}
+.highlight .s2 {
+    color: #e31b37 !important;
+}
+
+
+#search-results .context {
+    font-family: Roboto,Arial,sans-serif;
+}
+.rst-content .highlighted {
+    font-family: Roboto,Arial,sans-serif;
+  
+}
+footer {
+    font-family: Roboto,Arial,sans-serif;
+    color: #fcfcfc;
```

Comment:
Could do `font-size: 0px` instead of changing the color to white. Little nicer since your cursor won't go over it and allow you to highlight it and all that